lib/dynamics_crm/client.rb

```diff
@@ -141,7 +141,13 @@ def rollup(target_entity, query, rollup_type="Related")
         })
     end
 
-    def retrieve_multiple(entity_name, criteria=[], columns=[], operator=nil)
+    def retrieve_multiple_query(query_expression)
```

Comment:
I don't want to keep adding parameters to `retrieve_multiple`, they aren't named parameters and you have to remember the order.  I like building a `QueryExpression` by itself and then passing that in as a single parameter.